pywinauto/controls/common_controls.py

```diff
@@ -1028,23 +1028,23 @@ def rectangle(self, text_area_rect = True):
         del remote_mem
         return rect
     # Non PEP-8 alias
-    Rectangle = rectangle
+    ClientRect = client_rect

```

Comment:
Yes, the class name is internal (so that user shouldn't instantiate it by himself) but all the methods are pretty public. Maybe we need to make the class name public as well despite it's not a wrapper in the usual sense. OK, will discuss it later.